src/solc_0.8/polygon/child/land/PolygonLandBaseToken.sol

```diff
@@ -223,17 +223,17 @@ contract PolygonLandBaseToken is ERC721BaseToken {
         if (set) {
             if (!ownerOfAll) {
                 require(
-                    _owners[quadId] == uint256(sha256(abi.encodePacked(from))) ||
+                    _owners[quadId] == uint256(uint160(address(from))) ||
```

Comment:
Can we move `uint256(uint160(address)` to some sort of helper function ? (or maybe reuse one from openzeppelin ?)